presto-docs/src/main/sphinx/functions/bitwise.rst

```diff
@@ -28,4 +28,27 @@ Bitwise Functions
 
     Returns the bitwise XOR of ``x`` and ``y`` in 2's complement representation.
 
+.. function:: bitwise_shift_left_logical(x, shift, bits) -> bigint
```

Comment:
I personally found the `bits` parameter really confusing. If `x` is a `smallint`, am I supposed to know how many bits a `smallint` should have and provide the right value for `bits` to get a correct result?

Reply:
The bits convention is already a established with `bit_count`: https://prestodb.github.io/docs/current/functions/bitwise.html

I think, while confusing, it should be consistent; or, we deprecate the `bits` convention.  But for now I'd stick with consistency.